source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -40,45 +42,71 @@ guide, consider importing the following classes in your program:
    import static com.mongodb.client.model.mql.MqlValues.*;
    import static java.util.Arrays.asList;
 
-The examples use the ``Arrays.asList()`` method to create a list of
-aggregation operations that can be passed as a pipeline to the
-``aggregate()`` method.
-
-To construct an operation, first select a method that takes a value of
-type ``TExpression`` as a parameter, such as ``Projections.computed()``
-or ``Filters.expr()``. To reference document fields in an aggregation
-expression, obtain the "current" document being processed
-in the pipeline stage by calling the ``current()`` method. 
+To access document fields in an aggregation expression, you need to reference the
+document being processed by the aggregation pipeline. Use the
+``current()`` method to refer to this document.
 
-Since the aggregation operations are typed, you must specify the type of each
-field you reference using a typed variation of the ``getField()``
+To access the value of a field, you must use the appropriately typed
```

Comment:
[suggestion]
I'd consider combining this with the paragraph above about using `current()` since they are closely tied together. It would also make the previous paragraph feel less like it is free floating.

Reply:
agreed-- there are many short paragraphs in this section